Packages/GitHub/Sources/GitHubService/GitHubServiceVersion.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+public enum GitHubServiceVersion {
+  case dotCom
+  case enterprise(URL)
+
+  public enum Kind: String, CaseIterable {
+    case dotCom
```

Comment:
I wonder if there's an official name for "standard GitHub" that we can use or if that's just "GitHub.com".